quasar-utils/build.gradle

```diff
@@ -37,3 +39,18 @@ dependencies {
     testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:$junit_jupiter_version")
     testImplementation "org.assertj:assertj-core:$assertj_version"
 }
+
+tasks.withType(Test).configureEach {
+    javaLauncher = javaToolchains.launcherFor {
+        languageVersion = of(11)
+    }
```

Comment:
this will force the test execution to run under a different JVM (not the same JVM that is running Gradle) and I'm not sure users will be happy with that (I personally prefer configuring the gradle JVM myself instead of relying on Gradle toolchain black magic)

Reply:
Yes, because Quasar 0.8.x only runs on Java 11. However, the plugins themselves are built using Java 8.

Reply:
oh, right.. I initially thought it was for the tests in user's projects